PWGLF/Tasks/spectraCharged.cxx

```diff
@@ -48,6 +51,8 @@ struct chargedSpectra {
   Configurable<float> ptMinCut{"ptMinCut", 0.15f, "Pt min cut."};
   Configurable<float> ptMaxCut{"ptMaxCut", 10.f, "Pt max cut."};
 
+  bool fMCIsChargedPrimary{true};
```

Comment:
I would move this transient variable also into the varContainer and just call it `isChargedPrimary`.
The nomenclature with preceding `f` for member variables which we had in AliPhysics is no longer used in O2.
In structs (i.e. also in the tasks) there is no such rule for the naming of members, and for Classes one would add a preceding `m` instead of `f`.
Maybe later (when we have a few more variables) we can also use two varContainers, one for the transient event properties and one for the particle / track properties.